compose/cli/colors.py

```diff
@@ -1,8 +1,19 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+import enum
+
 from ..const import IS_WINDOWS_PLATFORM
 
+
+@enum.unique
+class AnsiMode(enum.Enum):
+    """Enumeration for when to output ANSI colors."""
+    never = "never"
+    always = "always"
+    auto = "auto"
```

Comment:
IMHO Something like the definition below would be better:
```python
@enum.unique
class AnsiMode(enum.Enum):
    """Enumeration for when to output ANSI colour's."""
    AUTO = "auto"
    NEVER = "never"
    ALWAYS = "always"

    def __init__(self, mode):
        self.mode = mode

    def use_tty(self, stream):
        return self is AnsiMode.ALWAYS or (self is AnsiMode.AUTO and stream.isatty())
```

Note the method `use_tty` that can be used after in the code
(Maybe that's not the best method name, feel free to change)